services/worker/src/worker/job_runners/config/parquet_metadata.py

```diff
@@ -95,6 +95,11 @@ def compute_parquet_metadata_response(
         ]
         if not parquet_file_items:
             raise ParquetResponseEmptyError("No parquet files found.")
+        content = config_parquet_best_response.response["content"]
+        if "features" in content and isinstance(content["features"], dict):
+            features = content["features"]  # config-parquet version<6 didn't have features
+        else:
+            features = None
```

Comment:
same comment:

maybe add a note to replace this part (features=None) later with an error. I understand you put it for backward-compatibility.